src/MongoDB.Analyzer/Core/Builders/BuilderExpressionProcessor.cs

```diff
@@ -106,6 +108,7 @@ public static ExpressionsAnalysis ProcessSemanticModel(MongoAnalyzerContext cont
             }
         }
 
+        BuildersResolveVariables.ResolveVariables(analysisContexts, buildersToAnalysisContextMap, semanticModel);
         var linqAnalysis = new ExpressionsAnalysis()
```

Comment:
Old copypaste issue, please rename linqAnalysis to builderAnalysis.

Reply:
Done